app/scripts/controllers/transactions/index.js

```diff
@@ -2316,6 +2316,7 @@ export default class TransactionController extends EventEmitter {
       ///: BEGIN:ONLY_INCLUDE_IN(blockaid)
       securityAlertResponse,
       ///: END:ONLY_INCLUDE_IN
+      simulationFails,
```

Comment:
Hey @segun, this is just a friendly reminder that we are decoupling metrics logic from controller, this change is a small one but I though worth mentioning here to clear future confusion if mine goes first.

https://github.com/MetaMask/metamask-extension/pull/21330
https://github.com/MetaMask/metamask-extension/pull/21426